### ISSUE TITLE
Unable to give `svg-img-alt` value to VictoryChart component

### ISSUE THREAD

**Issue by shankstee** [FormidableLabs/victory#2078]
Hello,

When running a A11y fast pass, it fails with the error listed below:
![image](https://user-images.githubusercontent.com/42012215/151679907-107f0876-deb6-4a4d-8e8c-c600956356b1.png)

When trying to add this `svg-img-alt` attribute to the VictoryChart component, there does not seem to be a prop available for this. 

Any help is welcomed, if I am missing something please let me know.

Thank you for your time!

**Comment by becca-bailey** [comment]
Hi @shankstee, thanks for letting us know! Victory currently doesn't pass user-defined props through to the base component, and we're working on a fix.

**Comment by Abe-Froman** [comment]
Hi @becca-bailey & @dlcartright , just wanted to check on the progress here. As I see #2082 has been closed, is there an timeline for when this might be included in a release? It doesn't look to be part of 36.3.1 but @shankstee and I would be thrilled if it were.

**Comment by Abe-Froman** [comment]
Thanks @becca-bailey completely understand. I'll see if we might be able to carve out some time to contribute for this, I just wanted to make sure we're not stomping on anyone's existing work if we were to do that.

**Comment by shankstee** [comment]
FYI, I am working on trying to resolve this issue now. I will hopefully have a PR out for it soon.

**Comment by shankstee** [comment]
@becca-bailey @dlcartright @Abe-Froman 
I've created a PR for this issue. Please let me know if I am missing anything or if your need me to make any further changes. Thanks for your time.

PR: https://github.com/FormidableLabs/victory/pull/2199
